util/chunk/compare.go

```diff
@@ -25,6 +25,16 @@ import (
 // CompareFunc is a function to compare the two values in Row, the two columns must have the same type.
 type CompareFunc = func(l Row, lCol int, r Row, rCol int) int
 
+// GetCompareTopNFunc get compare funcs by their string value For MAX()/Min() on ENUM and SET field type
```

Comment:
how about modify comment as follow:
// GetCompareTopNFunc gets a compare function for the field type used by TopNExec
// as EnumSet field type is sorted by name in TopNExec, but by value in SortExec